lua/lazyvim/plugins/extras/lang/tailwind.lua

```diff
@@ -28,7 +28,7 @@ return {
       },
       setup = {
         tailwindcss = function(_, opts)
-          local tw = require("lspconfig.server_configurations.tailwindcss")
+          local tw = require("lspconfig.configs.tailwindcss")
```

Comment:
Even before https://github.com/neovim/nvim-lspconfig/pull/3330, the documented way to use configs from nvim-lspconfig has always been:

```suggestion
          local tw = require'lspconfig'.tailwindcss
```

so I wasn't expecting https://github.com/neovim/nvim-lspconfig/pull/3330 to break anything.

I will add a shim to avoid breakage, but was there a reason to not use `require'lspconfig'.tailwindcss` (it would have avoided the breakage) ?

Reply:
My guess would be to get access to the `default_config.filetypes` table which is used later on. It's not for setting up the LSP server. Maybe `require'lspconfig'.tailwindcss.document_config` could be used instead? That way later on `tw.default_config.filetypes` will be accessible. Maybe there's a better way?

Reply:
Indeed. `default_config` is not available when getting it with `require'lspconfig'.tailwindcss`

Reply:
Ok. I added `config_def` for that purpose in https://github.com/neovim/nvim-lspconfig/pull/3335 , and that's now documented.